src/javascript/app/pages/trade/contracts.jsx

```diff
@@ -127,7 +124,7 @@ class Contracts extends React.Component {
                     ref={this.saveRef.bind(null, 'wrapper')}
                 >
                     <div className={`mobile_close invisible ${open && is_mobile ? '' : 'disabled'}`}>
-                        <span>{localize('Select Trade Type')}</span>
+                        <span>{it.L('Select Trade Type')}</span>
```

Comment:
This is a client side component. `it.L()` is only available on `src/templates/` folder for server side rendering.

Same applies to the changes on [`markets.jsx`](https://github.com/binary-com/binary-static/pull/4529/files#diff-7a3d234a678624c37405926dfebf6429).